R/main.R

```diff
@@ -11,6 +11,37 @@ check_drjacoby_loaded <- function() {
   message("drjacoby loaded successfully!")
 }
 
+#------------------------------------------------

```

Comment:
Is there a reason why we need to access the data like this?
Using `data()` with this as part of the package would mean we don't need this extra function.